src/Parachain.sol

```diff
@@ -134,9 +146,12 @@ abstract contract Parachain {
             _parachain.palletInstance, // pallet index within runtime
             hex"10", // fixed call index within pallet: 16
             _disputeId, // dispute id
-            uint8(_outcome) // outcome
```

Comment:
How is it ensured that this will be `uint8` if no longer cast as such?

Reply:
Why do you need to make it a uint8 abi.encodepacked packs everything togetgr

Reply:
I guess a test will be needed to determine the use